### ISSUE TITLE
Cannot update QDK for Visual Studio Code

### ISSUE THREAD

**Issue by tpetchel** [microsoft/qsharp-compiler#427]
**Describe the bug**

The [QnA](https://marketplace.visualstudio.com/items?itemName=quantum.quantum-devkit-vscode&ssr=false#qna) for the extension sends me here. Apologies if this is a duplicate.

I'm unable to update the QDK on macOS. I've tried uninstalling and reinstalling; no dice. 

**To Reproduce**

1. Install an earlier version of the QDK. I have 0.2.118063001 (yes, from June 2018).
1. Try to install a newer version. 
  1. Uninstall/reinstall procedures don't give you a newer version.
  1. Automatic updates don't pull in anything new.

The more information you can provide, the more likely someone will be successful at reproducing the issue and finding a fix.

**Expected behavior**

Automatic update should pull the QDK extension. I should also be able to upgrade.

**Screenshots**

**System information**

- macOS 10.13.6
- QDK 0.2.118063001
- Visual Studio Code 1.25.0

**Additional context**

As a workaround, I'm able to `dotnet new -i Microsoft.Quantum.ProjectTemplates --update-check` to get the latest project templates. From there, I can `dotnet new` a new Q# project. But I'm working on Microsoft Learn materials to help folks get started. The flow we prescribe uses the command palette, which isn't available in older QDK versions.

**Comment by bettinaheim** [comment]
@tpetchel What exactly do you mean by "reinstalling"? Did you download the newest extension and install that? Whether or not extensions in general are automatically update is a setting in VS Code.

**Comment by cgranade** [comment]
@tpetchel It just occurred to me, but you've listed that you're using version 1.25.0 (June 2018) of VS Code. As of version 0.3 of the Quantum Development Kit extension, version 1.31.0 (January 2019) of VS Code is required; can you update VS Code and see if that resolves the issue? Thanks for your patience!

**Comment by tpetchel** [comment]
@bettinaheim - by "reinstall", I meant I clicked Uninstall and then Install again.

@cgranade - great catch! So strange that I was running such an old version for so long. Whenever I clicked "Check for updates", Code always told me I was running the latest version. I downloaded and installed the latest Code package, and all is well now! I'm now able to update the extension.

![image](https://user-images.githubusercontent.com/7142397/82130973-1fe88c80-9785-11ea-80b4-b4a7771b5fca.png)

I figured it was something silly like that. Thanks to you both!
